Adds the NT Heavy Salvage Vessel as a new Ruin

CollectionOfBones128 opened the pull request:
## About the PR
Adds the NT Heavy Salvage Vessel, an SS13 White Ship, as a new Ruin.
![](https://wiki.tgstation13.org/images/b/bc/NT_Heavy_Salvage_Vessel.png)
(https://wiki.tgstation13.org/White_Ship)

## Why / Balance
We could use more variety in Ruins.
A [multichoice poll](https://discord.com/channels/1139716325627932682/1293447943696678944/1390617889123340381) was held, and this one entered an early lead. I had a real itch to map something and try out the image to map tool, so I decided to try my hand at making it

## Technical details
Adds the folder Maps/_Harmony/Ruins
Adds whiteship_heavysalvage.yml to said folder

Modifies Resources/Prototypes/Entities/Stations/base.yml to allow the new ruin to spawn

## Media
![Screenshot 2025-07-04 122358](https://github.com/user-attachments/assets/f5de1d62-3149-4f6c-aa57-49fe227cd312)
![Screenshot 2025-07-04 125545](https://github.com/user-attachments/assets/77cb710f-20cb-4954-a4e7-6fb72de5463b)

## Requirements
- [X] I have tested all added content and changes.
- [X] I have added media to this PR or it does not require an ingame showcase.

**Changelog**
:cl:
- add: Adds a new Ruin, the NT Heavy Salvage Vessel

CollectionOfBones128 commented:
> I really like it!
> 
> My one concern is that its very well put together for a ruin, and could probably use a bit more wear and tear? At the moment it seems like you could slap a shuttle console on and wabam, full shuttle ready to go.

It'll also need gyro, thrusters moved, and actual power.

I thiiink it should be a project thats about the same amount of effort as making a shittle from scratch? So on that front, it shouldn't affect the pace?

Its defo something I thought about though.

raccoonio reviewed (changes requested):
as said prior in comments here, this ruin is seemingly not in ruin. ruins are meant to be desolate, destroyed things that salvagers are meant to pillage, not fly. 
if they really want to fly it, they'll have to put in some actual work, rather than just slapping on a gyro and shuttle computer. punch some holes into this thing please

CollectionOfBones128 commented:
>this ruin is seemingly not in ruin. ruins are meant to be desolate, destroyed things that salvagers are meant to pillage

I disagree fundementally.

See upstream ruins:

- Biodome Satellite
- DJ Station
- Syndicate Dropship
- Whiteship Ancient

CollectionOfBones128 commented:
> if they really want to fly it, they'll have to put in some actual work, rather than just slapping on a gyro and shuttle computer. punch some holes into this thing please

Whiteship Ancient is a fully intact Shuttle shell.
Syndicate Dropship is just a fully functional shuttle.

To get this to work, you need to get and add components, move thrusters around, actually sort the power situation, & I think also reconfigure the command area doors

raccoonio commented:
i took a bit longer than i should've responding back to this, but here it is:

- the shuttle has no atmosphere, despite it being completely sealed. please run ``fixgridatmos [gridID]`` to fix this
- although the lower middle external airlocks bolts on opening, the other airlocks don't (these being the mining airlocks and windoors). please fix this
- on the topic of the airlocks, the outer airlocks are not salvage locked. this makes it easy for passengers to enter, despite the context of this shuttle being strictly for salvage. please fix this
- the captain's room has no reward for the salvagers. please give them some stuff, like money, retro laser, vel cetera
- the shuttle has *alot* of air alarms. you could replace a majority of them with random (preferably legal) poster spawners!
- there are also some firelocks that are a bit redundant, like the one leading to the captains room and the dorm rooms. these places don't need firelocks, since it just traps the inhabitants inside

CollectionOfBones128 commented:
> i took a bit longer than i should've responding back to this, but here is some feedback:

Okay, fixed all windoors that need access & replaced the airlocks with the locked ones

Made all external facing airlocks bolt

Added gold & money to the secure locker in the caps room

Replaced the air and fire alarm system to one each, only put firelocks in spaces adjacent to the central chamber.

Put posters up

Gonna save this, then do the atmos command and hope it works

FluffMe commented:
> can't tell if thats a heisentest or if I broke something with the air alarm

Rerun failed: https://github.com/ss14-harmony/ss14-harmony/actions/runs/16542677728/job/46788420129?pr=829

Doing another one. Run tests locally to confirm

CollectionOfBones128 commented:
yeah looks like I broke something with the atmos devices fuuuck

ilovehans10 commented:
This looks like a ton of fun to salvage! This is also apparently how I find out that SS13 has mi-go and god damn do I wish we had them too now.
